features/service_catalog_tsb/tsb.feature

```diff
@@ -12,51 +12,3 @@ Feature: Template service broker related features
     Then the output should contain "template-service-broker"
     Given cluster service classes are indexed by external name in the :csc clipboard
     Then the expression should be true> cb.csc.values.find {|c| c.cluster_svc_broker_name == "template-service-broker"}
-
-
-  # @author zitang@redhat.com
-  # @case_id OCP-14477
```

Comment:
Test case for 3.x only.
![Screenshot from 2021-09-16 17-25-54](https://user-images.githubusercontent.com/4716987/133587220-e27d388d-0615-4dbb-94be-08bc600aebee.png)